lib/calculate_format.rb

```diff
@@ -40,8 +40,13 @@ def any_of_format?(format)
     @cluster.ht_items.any? { |ht| item_format(ht) == format }
   end
 
-  def cluster_has_item_with_enum_and_same_ht_bib_key?(ht_item)
+  def record_has_any_enum?(ht_item)
     @cluster.ht_items.any? do |ht|
+      # TODO: reconsider what happens if n_enum is nil
```

Comment:
I added an issue https://hathitrust.atlassian.net/browse/DEV-1531 to address this more broadly